pkg/convert/json_ngxtranslate_test.go

```diff
@@ -0,0 +1,129 @@
+package convert
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.expect.digital/translate/pkg/model"
+)
+
+func Test_fromNgxTranslate(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name       string
+		serialized []byte
+		expected   model.Messages
+		wantErr    bool
+	}{
+		{
+			name:       "Not nested",
+			serialized: []byte(`{"message":"example"}`),
+			expected: model.Messages{
+				Messages: []model.Message{
+					{
+						ID:      "message",
+						Message: "example",
+					},
+				},
+			},
+			wantErr: false,
+		},
+		{
+			name:       "Nested normally",
+			serialized: []byte(`{"message":{"example":"message1"}}`),
+			expected: model.Messages{
+				Messages: []model.Message{
+					{
+						ID:      "message.example",
+						Message: "message1",
+					},
+				},
+			},
+			wantErr: false,
+		},
+		{
+			name:       "Nested with dot",
+			serialized: []byte(`{"message.example":"message1"}`),
+			expected: model.Messages{
+				Messages: []model.Message{
+					{
+						ID:      "message.example",
+						Message: "message1",
+					},
+				},
+			},
+			wantErr: false,
+		},
+		{
+			name:       "Nested mixed",
+			serialized: []byte(`{"message.example":"message1","message":{"example":"message1"}}`),
+			expected: model.Messages{
+				Messages: []model.Message{
```

Comment:
Identifiers are unique. It should not be possible to have two messages with the same ID.